tests/unit/CMakeLists.txt

```diff
@@ -114,10 +114,12 @@ bf_test_configure_non_build_srcs(tests_unit
         ${bpfilter_shared_srcs}
 )
 
+# cmocka needs PIE disabled to be able to auto-discover tests.
```

Comment:
It's not exactly CMocka, but the test harness performing autodiscovery (https://github.com/facebook/bpfilter/blob/main/tests/unit/harness/elf.c).